src/main/java/mate/hwdao/dao/impl/jdbc/ManufactureDaoJdbcImpl.java

```diff
@@ -0,0 +1,112 @@
+package mate.hwdao.dao.impl.jdbc;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.hwdao.dao.ManufacturerDao;
+import mate.hwdao.lib.Dao;
+import mate.hwdao.model.Manufacturer;
+import mate.hwdao.util.ConnectionUtil;
+
+@Dao
+public class ManufactureDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacture (name, country, isexist) VALUES (?, ?, ?)";
```

Comment:
```suggestion
        String query = "INSERT INTO manufacture (name, country) VALUES (?, ?)";
```
rename column according to suggested naming and use default value in db

Reply:
> at i left before

i din't get all in first time. sorry.